nerf/utils.py

```diff
@@ -376,6 +378,10 @@ def train_step(self, data):
         
         # encode pred_rgb to latents
         # _t = time.time()
+        # with profile(activities=[
```

Comment:
For the profiler maybe we can set a flag to enable/disable as we might need to do similar profiling if we start to change nerf module, SD new versions

Reply:
Thanks Vishal! I'll create another PR with those changes